CHANGELOG

```diff
@@ -1,6 +1,9 @@
 3.7.0 (2022-04-18):
     * Feature: support searching metadata with the class string of an entity
 
+3.7.0 (2023-04-19):
```

Comment:
v3.7.0 has already been released, can you update the changelog ?

Reply:
Maybe you will create a release with other MR, isn't it ?

Reply:
Right ! We merged https://github.com/ccmbenchmark/ting/pull/64 but haven't made a release yet.